Essentials/Commands/GridModule.cs

```diff
@@ -65,6 +66,72 @@ public void SetOwner(string gridName, string playerName)
             });*/
         }
 
+        [Command("ejectall", "Ejects all Players from given grid.")]
+        [Permission(MyPromoteLevel.SpaceMaster)]
+        public void Eject(string gridName = null) 
+        {
+            ConcurrentBag<MyGroups<MyCubeGrid, MyGridMechanicalGroupData>.Group> gridGroups;
+
+            if (gridName == null) 
+            {
+                if(Context.Player == null) 
+                {
+                    Context.Respond("The console always has to pass a gridname!");
+                    return;
+                }
+
+                IMyCharacter character = Context.Player.Character;
+
+                if (character == null) 
+                {
+                    Context.Respond("You need to spawn into a character when not using gridname!");
+                    return;
+                }
+
+                gridGroups = GridFinder.FindLookAtGridGroupMechanical(character);
+
+                if (gridGroups.Count == 0) 
+                {
+                    Context.Respond("No grid in your line of sight found! Remember to NOT use spectator!");
+                    return;
+                }
+            } 
+            else 
+            {
+                gridGroups = GridFinder.FindGridGroupMechanical(gridName);
+
+                if (gridGroups.Count == 0) 
+                {
+                    Context.Respond($"Grid with name '{gridName}' was not found!");
+                    return;
+                }
+
+                if (gridGroups.Count > 1) 
+                {
+                    Context.Respond($"There were multiple grids with name '{gridName}' to prevent any mistakes this command will not be executed!");
+                    return;
+                }
+            }
+
+            var group = gridGroups.First();
+
+            foreach(var node in group.Nodes) 
+            {
+
```

Comment:
Only use blank lines when separating logical sections within a block and keep it to one line of whitespace at most